pkg/driverbuilder/builder/debian.go

```diff
@@ -36,20 +36,24 @@ func (v debian) Script(c Config) (string, error) {
 	}
 
 	var urls []string
-    if c.KernelUrls == nil {
-        var kurls []string
-        kurls, err = fetchDebianKernelURLs(kr)
-        if err != nil {
-            return "", err
-        }
-        urls, err = getResolvingURLs(kurls)
-    } else {
-        urls, err = getResolvingURLs(c.KernelUrls)
-    }
-    if err != nil {
-        return "", err
-    }
-	if len(urls) < 2 {
+	if c.KernelUrls == nil {
+		var kurls []string
+		kurls, err = fetchDebianKernelURLs(kr)
+		if err != nil {
+			return "", err
+		}
+		urls, err = getResolvingURLs(kurls)
+	} else {
+		urls, err = getResolvingURLs(c.KernelUrls)
+	}
+	if err != nil {
+		return "", err
+	}
+	// We need:
+	// kernel devel
+	// kernel devel common
+	// kbuild package
+	if len(urls) < 3 {
```

Comment:
We already check that all of these 3 are fetched, see:
* kernel-headers: https://github.com/falcosecurity/driverkit/blob/master/pkg/driverbuilder/builder/debian.go#L213
* kernel-common: https://github.com/falcosecurity/driverkit/blob/master/pkg/driverbuilder/builder/debian.go#L228
* https://github.com/falcosecurity/driverkit/blob/master/pkg/driverbuilder/builder/debian.go#L259
The latter check if at least 2 string submatch can be found, than uses `match[1]`.